README.md

```diff
@@ -38,6 +38,7 @@ $ omf install https://github.com/jhillyerd/plugin-git
 | gban         | `git branch -a -v --no-merged`                       |
 | gbda         | delete all branches merged in current HEAD           |
 | ggsup        | git set upstream to origin/_current-branch_          |
+| gbage        | `git for-each-ref --sort=committerdate refs/heads/ --format='%(HEAD) %(color:yellow)%(refname:short)%(color:reset) - %(color:red)%(objectname:short)%(color:reset) - %(contents:subject) - %(authorname) (%(color:green)%(committerdate:relative)%(color:reset))'` |
```

Comment:
I don't think this description is easily readable, let's just summarize what it does.

Reply:
My local repo was indeed old and I didn't noticed I could simply put a description there => Done!